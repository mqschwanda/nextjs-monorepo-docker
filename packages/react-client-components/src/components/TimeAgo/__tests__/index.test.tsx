import { render } from '@testing-library/react';
import { TimeAgo } from '@';

describe('@mqs/react-client-components', () => {
  describe('components', () => {
    describe('<TimeAgo />', () => {
      it('renders', () => {
        const testId = TimeAgo.name;

        const { getByTestId, asFragment } = render((
          <TimeAgo
            date={new Date()}
            testId={testId}
          />
        ));

        const element = getByTestId(testId);
        expect(element).toBeTruthy();

        expect(asFragment()).toMatchSnapshot();
      });
    });
  });
});
