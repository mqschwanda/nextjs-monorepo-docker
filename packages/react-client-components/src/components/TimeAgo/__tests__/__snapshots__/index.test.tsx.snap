// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@mqs/react-client-components components <TimeAgo /> renders 1`] = `
<DocumentFragment>
  <span
    data-testid="TimeAgo"
  >
    just now
  </span>
</DocumentFragment>
`;
